doc/source/train/getting-started-pytorch-lightning.rst

```diff
@@ -23,7 +23,7 @@ For reference, the final code is as follows:
     from ray.train.torch import TorchTrainer
     from ray.train import ScalingConfig
 
-    def train_func(config):
```

Comment:
Don't show config in the first place, since we didn't specify `train_loop_config` in `TorchTrainer`. Users will be confused about where to put the `train_func` arguments.